packages/web/src/pages/settings-page/components/desktop/DeveloperApps/DeleteAppConfirmationPage.tsx

```diff
@@ -79,26 +77,23 @@ export const DeleteAppConfirmationPage = (
     <div>
       <h4 className={styles.header}>{name}</h4>
       <p className={styles.confirmation}>{messages.confirmation}</p>
-      <ModalFooter className={styles.deleteAppFooter}>
+      <ModalFooter css={{ paddingBottom: `0 !important` }}>
```

Comment:
important 🤮 
![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExN3B5bWc4cDR2dzF6eW9zaTM5N2p2OWl2eTg0bHJ4aGlxeWpxa3IxNSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/l41Ye7i203TfEADYs/giphy.gif)

Reply:
reason for that is that css modules override :/ we can update the modalfooter use emotion though!